db-shared/src/main/java/com/splicemachine/db/iapi/reference/GlobalDBProperties.java

```diff
@@ -136,4 +136,9 @@ public String getInformation() {
             new PropertyType("splice.execution.newMergeJoin",
                     "use new merge join. possible values: on, off, forced",
                     new Validator.MultipleOptions(new String[]{"on", "off", "forced"}));
+
+    public static PropertyType COST_MODEL =
+            new PropertyType("costModel",
+                             "cost model version, usable versions are v1 and v2",
```

Comment:
instead of "usable" I would write "available"

Reply:
Right, that's better. Changed to "available" in the latest commit.